conda/environment-dev.yml

```diff
@@ -23,8 +23,6 @@ dependencies:
     - pytest
     - pytest-cov
     - pytest-runner
-    - recommonmark
```

Comment:
The dependency list can be minimized to
```
dependencies:
  - python >=3.7,<3.9
  - pip
  - rdkit >=2020.03.1
  - pip:
    - ..[dev]
```
So we dont repeat the dev deps.

Reply:
Good idea! I'll try that.

Reply:
Gave some issues with ``python setup.py test`` which I will now try to remove/replace...